eslint-default.js

```diff
@@ -4,42 +4,45 @@ module.exports = {
     project: ["tsconfig.json"],
     ecmaVersion: 2018, // Allows for the parsing of modern ECMAScript features
     sourceType: "module", // Allows for the use of imports
-    ecmaFeatures: {
-      jsx: true, // Allows for the parsing of JSX
-    },
   },
-  plugins: ["@typescript-eslint", "sonarjs"],
+  plugins: ["@typescript-eslint", "sonarjs", "unused-imports"],
   extends: [
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:@typescript-eslint/recommended-requiring-type-checking",
-    "plugin:react/recommended",
-    "plugin:sonarjs/recommended",
     "plugin:eslint-comments/recommended",
-    "prettier", // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
+    "plugin:jest/recommended",
+    "plugin:sonarjs/recommended",
     "plugin:prettier/recommended", // Enables eslint-plugin-prettier and displays prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+    "prettier", // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
   ],
   rules: {
     curly: ["error", "all"],
+    "no-console": "warn",
     "no-unneeded-ternary": "error",
-    "no-redeclare": "off", // warns on typescript function overrides
```

Comment:
Máme ověřeno, že je možné tohle bezpečně vypnout? Dříve to mělo nějaké false-positives.

Reply:
Vypadá to, že tohle pravidlo je dupicitní s typescriptem.